ubc-wp-vote.php

```diff
@@ -23,11 +23,24 @@
 define( 'UBC_WP_VOTE_VERSION', '0.0.1' );
 define( 'UBC_WP_VOTE_PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
 define( 'UBC_WP_VOTE_PLUGIN_URL', plugin_dir_url( __FILE__ ) );
+define( 'UBC_WP_VOTE_DEFAULT_RUBRICS', array( 'Upvote', 'Downvote', 'Rate' ) );
```

Comment:
This is not really considered a best practice. Define statements are more like "switches". They're designed to be very simple - things like booleans or basic strings.

We really want `UBC_WP_VOTE_DEFAULT_RUBRICS` and `UBC_WP_VOTE_POST_TYPES_TO_EXCLUDE`  to be class constants, I think.